src/main/java/org/fundacion/jala/converter/models/facade/ExtractFacade.java

```diff
@@ -42,6 +43,7 @@ public static void getTextExtract(final ExtractTextParameter extractTextParamete
 
     /**
      * Extracts metadata from file.
+     *
```

Comment:
Dots are missing at the end of the java doc

Reply:
ok